src/mongo/db/mongod_options.cpp

```diff
@@ -62,6 +62,11 @@ using std::string;
 MongodGlobalParams mongodGlobalParams;
 
 extern DiagLog _diaglog;
+extern int cursorTimeoutMillis;
```

Comment:
This is exactly how the system works Christian and I worked on this before he left. For anything in setParameter you will find it calls a static functin. That ends up mapping the var to a global memory space and needs to have extern  to expose it to the code.

No that  would break how the export server Param system works in mongo